pkg/cmd/pr/shared/commentable.go

```diff
@@ -225,6 +232,50 @@ func updateComment(commentable Commentable, opts *CommentableOptions) error {
 	return nil
 }
 
+func deleteComment(commentable Commentable, opts *CommentableOptions) error {
+	comments := commentable.CurrentUserComments()
+	if len(comments) == 0 {
+		return errNoUserComments
+	}
+
+	lastComment := &comments[len(comments)-1]
```

Comment:
No need to grab a pointer:

```suggestion
	lastComment := comments[len(comments)-1]
```